translations/ja.yml

```
@@ -216,16 +217,21 @@ src:
             '2131015997': プロフィール画像
         Navigation:
           '165162143': アセット
+          '541301752': ワークスペース一覧
           '2057458630': プラグイン
-          '2611517824': ワークスペース
-          '3198092477': アカウント
-          '3503703629': プロジェクト一覧
+          '3198092477': 'アカウント'
+          '3336263640': プロジェクト一覧
           '3833270271': データセット
         Project:
           ArchivedMessage:
             '1354606874': 通知
             '4112013314': >-
               プロジェクトがアーカイブされました。アーカイブ済みプロジェクトの設定を変更する場合は、プロジェクトのアーカイブを解除してから変更可能です
+          BasicAuthSection:
+            '256145410': ''
+            '2237029987': ''
+            '2663586870': ''
+            '3076470241': ''
```

```suggestion
            '256145410': ベーシック認証を有効化
            '2237029987': パスワード
            '2663586870': ユーザー名
            '3076470241': ベーシック認証
```

Thanks for helping my fault. 🙇

---

sure